components/apimgt/org.wso2.carbon.apimgt.cleanup.service/src/main/java/org/wso2/carbon/apimgt/cleanup/service/OrganizationPurgeDAO.java

```diff
@@ -206,6 +212,245 @@ public void deleteKeyManagerConfigurationList(List<KeyManagerConfigurationDTO> k
+            connection.setAutoCommit(false);
+            try (PreparedStatement preparedStatement = connection.prepareStatement(query)) {
+                preparedStatement.setString(1, organization);
+
+                preparedStatement.executeUpdate();
+                connection.commit();
+            } catch (SQLException e) {
+                try {
+                    connection.rollback();
+                } catch (SQLException ex) {
+                    log.error("Failed to rollback remove pending application creation workflows for organization: "
+                            + organization, ex);
+                }
+                handleException("Error occurred while removing application creation workflows for organization: "
+                        + organization, e);
+            }
+        } catch (SQLException e) {
+            handleException("Error occurred while removing application creation workflows for organization: " +
+                            organization, e);
+        }
+    }
+
+    /**
+     * Remove pending Application Registrations
+     *
+     * @param organization Organization
+     * @throws APIManagementException when failed to delete pending application registrations
+     */
+    public void deletePendingApplicationRegistrations(String organization) throws APIManagementException {
+        String query = OrganizationPurgeSQLConstants.REMOVE_PENDING_APPLICATION_REGISTRATIONS;
+
+        try (Connection connection = APIMgtDBUtil.getConnection()) {
+            connection.setAutoCommit(false);
+            try (PreparedStatement preparedStatement = connection.prepareStatement(query)) {
+                preparedStatement.setString(1, organization);
+                preparedStatement.executeUpdate();
+
+                connection.commit();
+            } catch (SQLException e) {
+                try {
+                    connection.rollback();
+                } catch (SQLException ex) {
+                    log.error("Failed to rollback remove pending application registrations for organization: "
+                            + organization, ex);
+                }
+                handleException(
+                        "Error while deleting pending application registrations for organization: " + organization, e);
+            }
+        } catch (SQLException e) {
+            handleException("Error while deleting pending application registrations for organization: " + organization,
+                    e);
+        }
+    }
+
+    /**
+     * Deletes Applications along with subscriptions, keys and registration data
+     *
+     * @param organization Organization
+     * @throws APIManagementException if failed to delete applications for organization
+     */
+    public void deleteApplicationList(String organization) throws APIManagementException {
+
+        String getConsumerKeyQuery = OrganizationPurgeSQLConstants.GET_CONSUMER_KEYS_OF_APPLICATION_LIST_SQL;
+
```

Comment:
Don't put blank lines in these places. Only place if there is a logical differences